cmd/operator-sdk/olm/status.go

```diff
@@ -34,6 +34,7 @@ func newStatusCmd() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().StringVar(&mgr.Namespace, "olm-namespace", olm.DefaultNamespace, "OLM Namespace")
```

Comment:
This should be called in `Manager.AddToFlagSet()`.

Reply:
If the flag is added to`Manager.AddToFlagSet()`, then its available for `olm install` and `uninstall` subcommands. I think we don't want that based on https://github.com/operator-framework/operator-sdk/pull/2613#discussion_r387864398. we just need it for `olm status` subcommand

Reply:
Hi @jeyaramashok,

I think we should follow the @estroz suggestion since it also allows us to install and uninstall the OLM in/from the specified namespace. However, you are right in his comment https://github.com/operator-framework/operator-sdk/pull/2613#discussion_r387864398 shows that his suggestions are contradicting. 

IHMO and previous understanding the --olm-namespace should be applied in all olm subcommands indeed the install and uninstall. Otherwise, how to uninstall the --olm if it not be in the  default namespace?

Reply:
I am inclined with @estroz's https://github.com/operator-framework/operator-sdk/pull/2613#discussion_r387864398 since we use manifest files from https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.14.1/olm.yaml

the uninstall also the same the manifest yaml files

Reply:
I agree. I checked that the install and uninstall are also setting the `namespace: olm`

Reply:
but if we want the capability to install/uninstall olm to any namespace ( which the user wants) , i could think of couple of ways to do it
- add in extra steps to override the namespace in those manifest files
- or have those objects coded inside operator-sdk. 
have to weigh pro/cons of this and think that's best to handle it in a seperate PR in future if there is a request for it